ui/README.md

```diff
@@ -0,0 +1,24 @@
+# TesLight UI
+
+## Requirements
+
+- [Node.js](https://nodejs.org/) (18.x.x)
+- [pnpm](https://pnpm.io/) (7.x.x)
```

Comment:
Since npm works as well, should we maybe add it? I think it might be easier for some user, since it already comes with Node. Any opinion on that?

Reply:
Yes, `npm` also works and can be used. I would prefer [pnpm](https://pnpm.io/).

It is very easy to install:

```bash
corepack enable
corepack prepare pnpm@latest --activate
```

A user shouldn't need this at all, right?

Reply:
I am personally fine with both. But we then should document it in the build guide. Building the frontend is required for the first-time setup. You then need to copy the frontend files to the MicroSD card.

Reply:
I understand. Would it be useful to make a built package available for download?

Reply:
Hmm, true... Maybe we just go that way... Then let's always add the built frontend to the realease. However we need to adjust the build guide then. Maybe we should describe both in there?

Reply:
I've added support for `npm`.